shared/menubar/index.js

```diff
@@ -30,25 +30,14 @@ export type MenubarProps = {
 
 class Menubar extends Component {
   props: MenubarProps;
```

Comment:
while making provision work i ran into the bug @gabriel saw where getting favorites would freak out